src/fread.c

```diff
@@ -785,7 +773,7 @@ static void parse_double_regular_core(const char **pch, double *target)
     }
     e += Eneg? -E : E;
   }
-  if (e<-350 || e>350) return;
+  if (e < -350 || e > 350) return;
```

Comment:
this one is particularly welcome 🙏

Reply:
Yeah, good one! This what Matt always mentioned when asked why he uses = vs <- in R